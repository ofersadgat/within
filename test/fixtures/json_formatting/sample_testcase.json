{"colors":["red","green","blue"],"person":{"name":"Bob","age": 20,"contact":{"email":"bob@example.com",},},"truthy": True,"falsy": False,"pi": 3.14159,}