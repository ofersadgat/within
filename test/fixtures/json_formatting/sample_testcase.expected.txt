"colors": 
[
	"red"
	"green"
	"blue"
]
"falsy": false
"person": 
{
	"age": 20
	"contact": 
	{
		"email": "bob@example.com"
	}
	"name": "Bob"
}
"pi": 3.14159
"truthy": true